src/main/java/org/folio/circulation/resources/ItemsInTransitResource.java

```diff
@@ -49,6 +52,8 @@
 
   private static final String ITEM_ID = "itemId";
   private final String rootPath;
+  private static final Comparator<InTransitReportEntry> IN_TRANSIT_REPORT_ENTRY_COMPARATOR =
```

Comment:
Why does the comparator need to be a static field?

Reply:
Good point. I wanted that the compactor was only one in the application.  However, now I think it is better to move it to getMany method. Moved

Reply:
Good point. I wanted that the compactor was only one in the application.  However, now I think it is better to move it to getMany method. Moved